### PR TITLE
Added fsharp.hrc (F#) and its record in proto.hrc

### PR DESCRIPTION
**Pull request by nightroman**

### CODE REVIEW

File: hrc/hrc/proto.hrc [hunk 1]
```diff
@@ -125,6 +125,10 @@
     <location link="base/csharp.hrc"/>
     <filename>/\.cs$/i</filename>
   </prototype>
+  <prototype name="fsharp" group="main" description="F#">
```
Comment (techtonik) [inline hrc/hrc/proto.hrc]: Is it really so popular? Maybe move it into `rare`?

Comment (nightroman) [inline hrc/hrc/proto.hrc]: This is up to you and this is not really important.

What about forth.hrc, kotlin.hrc, fortran.hrc, etc?

F# is the member of .NET family, so it is not much different from C# or VB.NET. It is also a scripting language, so *scripts* is also a candidate category. But *rare* is a bit weird.

Personally, I think it is a strange idea to categorize schemes by popularity.

Comment (nightroman) [inline hrc/hrc/proto.hrc]: Could you please give a clear answer? Were you interested in my opinion or telling me to move it to `rare` in order to get PR accepted?

Thank you for your cooperation.

Comment (techtonik) [inline hrc/hrc/proto.hrc]: Sorry, slipped off from my radar. I am thinking that introducing new languages to the list may shift automatic keyboard shortcuts, and also that the list of main languages should be updated with http://www.tiobe.com/tiobe-index/

Kotlin is not counted there. Well, ok. I guess it is an issue for another bug report for release. )

**Comment by nightroman** [comment]
Thank you!

A note about tiobe-index. I have been watching it for a few years, and that is
what I think: it sucks really hard. Of course, it shows *something* and may be
used to some extent.

I think the modern and the most trustful way is comparison by number of
questions per language on Stack Overflow.

**Comment by techtonik** [comment]
They describe how do they count. You may try http://spectrum.ieee.org/static/interactive-the-top-programming-languages-2016 - should be more authoritative.

**Pull request merged**